pykern/pkunit.py

```diff
@@ -351,7 +351,8 @@ def pkre(expect_re, actual, flags=re.IGNORECASE + re.DOTALL):
         actual (object): run-time value
         flags: passed on to re.search [IGNORECASE + DOTALL]
     """
-    if not re.search(expect_re, actual, flags=flags):
+    from pykern import pkcompat
```

Comment:
This import can be at the top and should be.

Reply:
7918c4a

Never sure when to place at the top or in a function...

Reply:
It's when there is likely to be a dependency issue. pkcompat is specifically designed to not be a dependency problem. In general, at the top is the right default. In Sirepo, there is more complexity with circular imports, especially with all the dependencies, e.g. h5py.